Misc/NEWS.d/next/C API/2024-03-10-14-55-51.gh-issue-116560.x2mZaO.rst

```diff
@@ -0,0 +1,2 @@
+Add :c:func:`PyLong_Sign` function (former private :c:func:`!_PyLong_Sign`).
```

Comment:
This change also deserves entry in "What's new" `(Doc/whatsnew/3.13.rst)` in the "C API Changes" section.

Reply:
Thanks, done.